<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtClausula</class>
 <widget class="QMainWindow" name="QtClausula">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtClausula</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineaClausula"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Cargar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Cláusulas:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listaClausulas"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="botonBorrar">
          <property name="text">
           <string>Borrar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonLimpiarClaus">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="botonVerificarSatis">
          <property name="text">
           <string>Verificar Satisfacibilidad</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="botonDetener">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Detener</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Información</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textoInfo">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="botonLimpiarInfo">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSatis">
          <property name="font">
           <font>
            <family>Arial Black</family>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <widget class="QMenu" name="menuCl_usulas">
    <property name="title">
     <string>Cláusulas</string>
    </property>
    <addaction name="actionVerificar_Satisfacibilidad"/>
    <addaction name="actionSimplificar_Conjunto"/>
   </widget>
   <widget class="QMenu" name="menuLiterales">
    <property name="title">
     <string>Literales</string>
    </property>
    <addaction name="actionUnificar_literales"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuCl_usulas"/>
   <addaction name="menuLiterales"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionVerificar_Satisfacibilidad">
   <property name="text">
    <string>Verificar Satisfacibilidad</string>
   </property>
  </action>
  <action name="actionSimplificar_Conjunto">
   <property name="text">
    <string>Simplificar Conjunto</string>
   </property>
  </action>
  <action name="actionUnificar_literales">
   <property name="text">
    <string>unificar literales</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>agregarClausula()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineaClausula</sender>
   <signal>returnPressed()</signal>
   <receiver>QtClausula</receiver>
   <slot>agregarClausula()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonBorrar</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>eliminarClausula()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>404</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonLimpiarClaus</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>eliminarClausulas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonVerificarSatis</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>verificarSatisfacibilidad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonDetener</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>detenerResolucion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>579</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>botonLimpiarInfo</sender>
   <signal>clicked()</signal>
   <receiver>QtClausula</receiver>
   <slot>limpiarTexto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>494</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbrir</sender>
   <signal>activated()</signal>
   <receiver>QtClausula</receiver>
   <slot>abrirArchivo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGuardar</sender>
   <signal>activated()</signal>
   <receiver>QtClausula</receiver>
   <slot>guardarClausulas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionVerificar_Satisfacibilidad</sender>
   <signal>activated()</signal>
   <receiver>QtClausula</receiver>
   <slot>verificarSatisfacibilidad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>agregarClausula()</slot>
  <slot>eliminarClausula()</slot>
  <slot>eliminarClausulas()</slot>
  <slot>verificarSatisfacibilidad()</slot>
  <slot>detenerResolucion()</slot>
  <slot>limpiarTexto()</slot>
  <slot>abrirArchivo()</slot>
  <slot>guardarClausulas()</slot>
 </slots>
</ui>
